### ISSUE TITLE
More help with reviewers/committers/maintainers?

### ISSUE THREAD

**Issue by macnewbold** [pallets-eco/flask-debugtoolbar#208]
Firstly, thank you to all the contributors and maintainers who have made this tool available, I'm very grateful for it and use it regularly.

In looking at the release history, and the current list of issues and pull requests, it seems like there may be room for more help in addressing the issues and pull requests, so that there can be more frequent updates to the tool. As the related projects in the Flask ecosystem continue to move forward, it takes some work to maintain compatibility with newer versions, and it would be really nice to be able to get those updates into official releases rather than needing to resort to pinning to older versions or forking my own copy of the debug toolbar.

Is the team open to other volunteers who are interested in helping keep the updates happening on this project?

**Comment by jeffwidman** [comment]
Yes, absolutely.

I haven't worked much with Flask in 5+, so don't use this tool but just chip in maintenance to keep the project alive. Similarly, I don't think @nickjj actively uses it a lot either, but he's also a nice guy who chips in on maintenance.

So if you (or anyone else) is interested in helping maintain the project, can you me an email (available in my profile) explaining why you're interested and what (if any) is your background in open source? I'll add in the other maintainers to that email thread and we can figure out the right next steps.

**Comment by jeffwidman** [comment]
We've added @macnewbold and @greyli as maintainers. There's another fellow who expressed interest although still we need some more information before adding him, so that's an ongoing conversation.

@ddorian are you at all interested in ☝️ ? I've noticed you staying involved in the issue tracker over time. If so, shoot me an email and I'll loop in the other maintainers and we can chat about it a little more.

Overall though, we're probably in a healthier place at least for now, so I'm going to close this as resolved. 

We _are_ still open to folks who are interested, so if you see this, just because it's resolved doesn't mean you shouldn't express interest. Just that the project isn't completely unmaintained.
